### PR TITLE
Fixes some Anti-AdBlocks and popups

### PR DESCRIPTION
**Pull request by wout-codes**
Fixes some Anti-AdBlocks and pop-ups from several sites.
Use AdGuard for Windows, Mac or Android to reproduce popups and Anti-AdBlock

Links:
1. `https://coindice.win/`                                    -         Anti-AdBlock
2. `https://freebitcoin.win/home`                       -         shortly after you will be redirected to disable AdGuard (requires an account).
3. `https://cut-fly.com/OHjzi` - Popups and Anti-AdBlock
4. `https://www.gioialive.it/`   - Anti-AdBlock
5. `https://hanimesubth.com/` - Popups
6. `https://mega4up.com/lr0mwpg7q0va`        -          pop-ups at the last page after Captcha.
7. `https://diariodegoias.com.br/goiania-tem-mais-um-caso-de-falha-na-aplicacao-da-vacina-veja/` - Anti-AdBlock
8. `guideautoweb.com` - Anti-AdBlock
9. `burningseries.co` - Popups
10. `bs.to` - Popups
11. `https://searchenginereports.net/`   -  Placeholders/leftovers
12. `dailyvideoreports.net` - Anti-AdBlock
13. `https://eroticmv.com/young-nurses-in-love-1984/?asgtbndr=1` - Pop-ups
14. `https://readmanga.se/manga/the-beginning-after-the-end/` - Useless Ad-Server and Anti-AdBlock
15. `https://courses4u.info/` - Anti-AdBlock
16. `https://www.friendproject.net/home.php` - Anti-AdBlock (requires an account)

### CODE REVIEW

**Comment by Yuki2718** [comment]
Add `mega4up.com#%#//scriptlet('adjust-setTimeout', 'tick', '1000', '0.02')` to Annoyances > tweaks.txt.

**Comment by wout-codes** [comment]
> Add `mega4up.com#%#//scriptlet('adjust-setTimeout', 'tick', '1000', '0.02')` to Annoyances > tweaks.txt.

Thanks! I was on it.

**Comment by wout-codes** [comment]
@Alex-302 @zloyden @AdamWr Can you verify these rules?

**Review by Alex-302 — APPROVED** [review]

**Review by Alex-302 — COMMENTED** [review]
Please don't make changes for a lot of sites on one pull request. It may be never merged.

File: EnglishFilter/sections/foreign.txt [hunk 1]
```diff
@@ -8741,6 +8741,7 @@ mm-cg.com##.TW_UTtoy
 !------------------------------ Thai ---------------------------------
 !---------------------------------------------------------------------
 !
+*.gif$domain=hanimesubth.com,image
```
Comment (Alex-302) [inline EnglishFilter/sections/foreign.txt:8744]: can be added without content type as `.gif^$domain=hanimesubth.com`

Comment (Yuki2718) [reply EnglishFilter/sections/foreign.txt:8744]: Well, it makes sense at least on uBO. Anyway it's a tiny difference, not?

Comment (Alex-302) [reply EnglishFilter/sections/foreign.txt:8748]: How do you measure the impact on performance?

Comment (Yuki2718) [reply EnglishFilter/sections/foreign.txt:8748]: I don't rigorously measure, but gorhill benchmarks regularly and we uAssets maintainer generally care about tokenization. See gorhill's reply here: `https://www.reddit.com/r/uBlockOrigin/comments/lovx3z/what_is_the_cleanest_way_to_block_an_image_format/`
Ofc it is AG repo here, but some people including me use Base filter on uBO too ;)

Comment (wout-codes) [reply EnglishFilter/sections/foreign.txt:8748]: Yep, thanks @Yuki2718. I based on that because uBO isn't bad after all.

**Comment by wout-codes** [comment]
> Please don't make changes for a lot of sites on one pull request. It may be never merged.

@Alex-302 So should I open 16 pull requests?

**Pull request merged**